src/CurrencyInput/storybook/CurrencyInput.stories.tsx

```diff
@@ -35,6 +35,19 @@ DefaultFallback.args = {
 }
 
 export const Collection = CollectionPage.bind({})
+Collection.parameters = {
```

Comment:
addressed the reason for this in the PR description - imo this is a design decision on how to fix these and will follow up to flag to them 

skipping until then